scalableminds/webknossos
# Add finer control over interval range in timetracking view

This PR adds a finer control to the time interval in the timetracking view.

### URL of deployed dev instance (used for testing):
- https://finertimetrackingviewinterval.webknossos.xyz

### Steps to test:
- Trace a task.
- Open the timetracking view and select the last two hours.
- The requested and displayed interval should only show those two hours.

### Issues:
- fixes #4581

------
